spec/mollie/client_spec.rb

```diff
@@ -62,6 +62,42 @@
     end
   end
 
+  context 'methods' do
+    it 'returns a hash with methods' do
+      VCR.use_cassette('get methods list') do
+        client = Mollie::Client.new(api_key)
+        response = client.methods
+
+        expect(response['totalCount']).to eql 8
+        expect(response['data'].first['id']).to eql 'ideal'
+        expect(response['data'].first['description']).to eql 'iDEAL'
+
+        expect(response['data'].first['amount']['minimum']).to eql '0.55'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.